core/templates/dev/head/components/ratingsSpec.js

```diff
@@ -116,14 +116,6 @@ describe('Ratings from frequencies directive', function() {
       '4': 0,
       '5': 0
     })).toBe(undefined);
-
-    expect(ctrlScope.computeAverageRating({
```

Comment:
No one will remember to fix it. How about making the constant into a global one (at the oppia level) and swapping it out here?